CODING_STANDARDS.md

```diff
@@ -0,0 +1,3 @@
+# Coding Standards
+
+Except if stated otherwise, we follow the [Zeth coding standards](https://github.com/clearmatics/zeth/blob/master/CODING_STANDARDS.md).
```

Comment:
should the points about the template parameter ames (e.g. nppT, etc) also go in here?

Reply:
That's already part of the Zeth coding standards on `develop`, so I would not add it here too. There will be a tiny gap until `zeth/develop` is merged to `zeth/master` but I think that's fine.
Alternatively, I can point to the version of the doc on `develop` to remove any possible misunderstanding.

Reply:
Not vital, but just to be clear, I mean the use of `n` for the nested curve/scheme, `w` for the wrapper curve/scheme. Not the "lower-case + T for type parameters" rule.
The README contains a list of type parameter names used in the implementation, which seems more part of the coding standards.

No problem if you prefer to keep that in README - just explaining since your comment seemed to be about somethign else.

Reply:
Ah! Sorry, I didn't understand your point. I thought you referred to: https://github.com/clearmatics/zeth/pull/233#discussion_r461440710

Good point, I like this idea. I'll move the nested/wrapping thing in the coding standards

Reply:
Moved